benchflow-faban-client/src/main/java/cloud/benchflow/faban/client/commands/StatusCommand.java

```diff
@@ -61,9 +61,9 @@ private RunStatus status(FabanClientConfig fabanConfig)
         throw new FabanClientBadRequestException("Illegal status request");
       }
 
+      //Handle generic HTTP exceptions (TODO: determine the expected HTTP status from Faban, and validate we get that one)
       //TODO: check that the call to .handleEntity(..) actually returns the expected string
```

Comment:
Fixed in https://github.com/benchflow/benchflow/pull/547/commits/810d57c4e6381d5b7d63add3419e13a657ce21f7